src/Fields/field_tuples.jl

```diff
@@ -197,32 +197,21 @@ TracerFields(::NamedTuple{(), Tuple{}}, grid, bcs) = NamedTuple()
 #####
 
 """
-    PressureFields(grid, bcs::NamedTuple)
+    PressureField(grid, bcs::NamedTuple)
```

Comment:
Do we need this at all, or is it simpler to put it in the model constructor?

Reply:
If that's the only place that method is called, then I think better to put it directly in the model constructor. But I'm not sure that's the case.

Reply:
What's called in the constructor is

https://github.com/CliMA/Oceananigans.jl/blob/6c388411f7101c51d93c9982bd8851cd2e370c75/src/Models/NonhydrostaticModels/nonhydrostatic_model.jl#L176

with `pressure==nothing` by default. So I guess the reason we don't call `PressureField(grid, bcs)` directly is to make it easier for users to pass a `pressure` directly to the user-facing model constructor and override the default.